products/cloudscheduler/terraform.yaml

```diff
@@ -63,6 +63,8 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         custom_flatten: 'templates/terraform/custom_flatten/http_headers.erb'
         validation: !ruby/object:Provider::Terraform::Validation
           function: 'validateHttpHeaders()'
+      appEngineHttpTarget.appEngineRouting: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
we can actually skip this flatten and just add `ignore_read: true` in the api.yaml to appEngineRouting

Reply:
I'v tried it and run `bundle exec compiler -p products/cloudscheduler -v "ga" -e terraform -o $GOPATH/src/github.com/terraform-providers/terraform-provider-google`, but then the generated diff disappears.
Am I doing right? Should I do anything else?

```diff
diff --git a/products/cloudscheduler/terraform.yaml b/products/cloudscheduler/terraform.yaml
index 1afe4db4..f15df123 100644
--- a/products/cloudscheduler/terraform.yaml
+++ b/products/cloudscheduler/terraform.yaml
@@ -64,7 +64,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         validation: !ruby/object:Provider::Terraform::Validation
           function: 'validateHttpHeaders()'
       appEngineHttpTarget.appEngineRouting: !ruby/object:Overrides::Terraform::PropertyOverride
-        custom_flatten: templates/terraform/custom_flatten/cloudscheduler_job_appenginerouting.go.erb
+        ignore_read: true
       region: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
         ignore_read: true
diff --git a/templates/terraform/custom_flatten/cloudscheduler_job_appenginerouting.go.erb b/templates/terraform/custom_flatten/cloudscheduler_job_appenginerouting.go.erb
deleted file mode 100644
index 3bf9cf7a..00000000
--- a/templates/terraform/custom_flatten/cloudscheduler_job_appenginerouting.go.erb
+++ /dev/null
@@ -1,26 +0,0 @@
-<%# The license inside this block applies to this file.
-	# Copyright 2019 Google Inc.
-	# Licensed under the Apache License, Version 2.0 (the "License");
-	# you may not use this file except in compliance with the License.
-	# You may obtain a copy of the License at
-	#
-	#     http://www.apache.org/licenses/LICENSE-2.0
-	#
-	# Unless required by applicable law or agreed to in writing, software
-	# distributed under the License is distributed on an "AS IS" BASIS,
-	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-	# See the License for the specific language governing permissions and
-	# limitations under the License.
--%>
-// An `appEngineRouting` filed of CloudScheduler API only contains output-only `host` field,
-// so we set config values rather than api response to state.
-// https://cloud.google.com/scheduler/docs/reference/rest/v1/projects.locations.jobs#AppEngineRouting
-func flatten<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d *schema.ResourceData) interface{} {
-	if v == nil {
-		return nil
-	}
-	if targets := d.Get("app_engine_http_target").([]interface{}); len(targets) > 0 {
-		return targets[0].(map[string]interface{})["app_engine_routing"]
-	}
-	return nil
-}
```

Reply:
I've pushed the commit: 701c4aa.
Let's see what the magician will generate.

Reply:
I found this; let me drop last commit, and just add `ignore_read: true` **alongside** custom flatten.

https://github.com/GoogleCloudPlatform/magic-modules/blob/15e22ca4e55f952eb5b79f674cd0f2e950ac83c3/products/compute/terraform.yaml#L1230-L1236

Reply:
okay, I removed `ignore_read` line and modified comment to address the current situation
8ec185e